Cargo.toml

```diff
@@ -60,6 +61,7 @@ batch = ["alloc", "merlin", "rand_core"]
 legacy_compatibility = []
 pkcs8 = ["ed25519/pkcs8"]
 pem = ["alloc", "ed25519/pem", "pkcs8"]
+rand_core = ["dep:rand_core"]
```

Comment:
It is best to keep iterating explicitly all the features in Cargo.toml as it's easier to understand for downstream